### ISSUE TITLE
Lint/UnusedBlockArgument example code seem to be wrong.

### ISSUE THREAD

**Issue by siman-man** [bbatsov/rubocop#4155]
It is written in the [document](http://rubocop.readthedocs.io/en/latest/cops_lint/#lintunusedblockargument)

```ruby
# bad
do_something do |used, _unused|
  puts used
end

# good
do_something do |used, unused|
  puts used
end
```

run `rubocop`

```
$ rubocop unused_block_argument.rb 
Inspecting 1 file
W

Offenses:

unused_block_argument.rb:7:24: W: Unused block argument - unused. If it's necessary, use _ or _unused as an argument name to indicate that it won't be used.
do_something do |used, unused|
                       ^^^^^^

1 file inspected, 1 offense detected
```

actual, warning message show for good case.

**Comment by bbatsov** [comment]
Yeah, those should be inverted of course. I'll fix it.
